frontend/src/packages/articles/articles-api.package.ts

```diff
@@ -101,6 +101,20 @@ class ArticleApi extends HttpApi {
 
     return await response.json<ArticleWithAuthorType>();
   }
+
+  public async delete(id: number): Promise<ArticleWithAuthorType> {
+    const response = await this.load(
+      this.getFullEndpoint(ArticlesApiPath.$ID, { id: String(id) }),
+      {
+        method: 'DELETE',
+        contentType: ContentType.JSON,
+        payload: JSON.stringify(id),
```

Comment:
You have already passed id as param + taking it from params at BE
```suggestion
```

Reply:
If I leave just` contentType: ContentType.JSON`, I will get an error: Body cannot be empty when content-type is set to 'application/json'. I cannot also just remove contentType as it's required in HttpApiOptions. I thought of using PATCH instead of DELETE, cause we actually changing deleteAt field, but there is already PATCH method on the articles/id route. What should I do in this case?

Reply:
Then why can't you just make `contentType` optional, and in getHeaders attach it only if it's present